internal/manifests/manifestutils/labels.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 func IsFilteredSet(sourceSet string, filterSet []string) bool {
```

Comment:
do we need to compile the pattern here? Is it not enough to just match string on it?

Reply:
Theoretically, we shouldn't, but without doing that, the regexp match doesn't work as expected.

Reply:
What is the difference between them?